daml/Marketplace/Exchange.daml

```diff
@@ -1,7 +1,7 @@
 module Marketplace.Exchange where
 
-import qualified Marketplace.Listing as Listing
-import qualified Marketplace.Trading as Trading
+import Marketplace.Listing qualified as Listing
+import Marketplace.Trading.Service qualified as TradingService
```

Comment:
Do you know if there's any difference where you put the `qualified`, or is it the same? Just curious.

Reply:
Theres no difference as far as I can tell - I took it from spider but when searching around on the internet I didnt find anywhere where it mentions `import <module> qualified as <name>` syntax. I just think this version is easier to read alongside other modules which aren't `qualified` since they all being with `import <module>`